GameData/ROEngines/PartConfigs/S155_CH4.cfg

```diff
@@ -0,0 +1,66 @@
+PART
+{
+	module = Part
+	name = ROE-S155
+	author = Luci, capkirk
+	
+	RSSROConfig = true
+	ROESetEngineDefaults = LIQUID_ENGINE
+	
+	//	============================================================================
+	//	Update Below
+	//	============================================================================
+	
+	MODEL
+	{
+		model = ROEngines/Assets/TheBeastlyPig/XLR25
+		scale = 1, 1, 1 
+	}
+	
+	scale = 1.0
+	rescaleFactor = 0.66
+	node_stack_top = 0,0,0,0,1,0,2
+	node_stack_bottom = 0,-0.8,0,0,-1,0,2
+	node_attach = 0,0,0,0,1,0,2
+	// stack, srfAttach, allowStack, allowSrfAttach, allowCollision
+	attachRules = 1,1,1,1,0
+
+	//let engineConfig set these
+	title = S-155
+	manufacturer = abc
+	description = abc
+
+	tags = MiG-21, MiG-23, S-155, Dushkin, Ye-50, ye50, MiG
+
+	engineType = S155
+
+	//keep these here because the patchs don't always work otherwise?
+	cost = 0
+	entryCost = 0
```

Comment:
ROESetEngineDefaults should have %s and always work without these fields present now.  (If that isn't the case, fix in ROESetEngineDefaults)